content/integrations/pingdom.md

```diff
@@ -28,3 +28,10 @@ Pingdom events can be added by configuring the relevant [Integration Status Moni
 {{< get-metrics-from-git >}}
 
 
+## FAQ
+### Why updating user/password and saving the changes returns an error ?
+If you have the following error while trying to save your pingdom integration: 
+
+`“There was an issue while testing your Pingdom configuration: Not permitted for account type”`.
```

Comment:
Since the error is short, I think we can just make it the title (i.e. header) of the FAQ.